src/main/java/banking/database/Operations.java

```diff
@@ -0,0 +1,149 @@
+package banking.database;
+
+import banking.processing.Account;
+import java.sql.Connection;
+import java.sql.DriverManager;
+import java.sql.PreparedStatement;
+import java.sql.ResultSet;
+import java.sql.SQLException;
+import java.sql.Statement;
+
+public class Operations {
```

Comment:
When you work with a database table, your class responsible for queries should call `modelName`+`Dao`or`Repository`. It depends on the stack that you are working on. In this case, it is better to use `modelNameDao`.

Reply:
fixed